tests/framework/e2e/cluster.go

```diff
@@ -613,6 +603,27 @@ func (cfg *EtcdProcessClusterConfig) EtcdServerProcessConfig(tb testing.TB, i in
 		panic(fmt.Sprintf("Unknown cluster version %v", cfg.Version))
 	}
 
+	defaultValues := values(*embed.NewConfig())
```

Comment:
Could we add a validation for the edge case that non of version is supports overriding "experimental-snapshot-catchup-entries" but "experimental-snapshot-catchup-entries" is set?

Reply:
done